tslint.json

```diff
@@ -5,6 +5,7 @@
             true,
             "statements"
         ],
+        "no-trailing-whitespace": [true, "ignore-template-strings"],
```

Comment:
Could you revert this? I'd like to know about trailing whitespaces in template strings.